### DISCUSSION TITLE
Issue with Codespaces Storage Consumption

### DISCUSSION THREAD

**Discussion by am-IRh** [community/community#116842] [Codespaces]
Hello 
I hope you're all doing well. I'm reaching out to report an issue I've been encountering with Codespaces. For some reason, the storage usage of my Codespaces keeps increasing without any apparent cause, eventually leading to full storage consumption.

Even after deleting all previous Codespaces and ensuring that I have no active Codespaces, the problem persists. I would greatly appreciate it if anyone could shed some light on this issue and provide any insights or solutions.

Thank you for your assistance.

![IMG_20240405_034052.jpg](https://github.com/community/community/assets/78879226/3eae952b-f6ed-4103-9433-fd6014529a2c)

![IMG_20240405_034108.jpg](https://github.com/community/community/assets/78879226/176e5abf-10d8-4d73-8354-fc1cc9f3919f)

**Comment by Center-Deal** [comment]
Hi. Same issue here. I haven't used Codespaces in months, certainly never anything even close to the limit, and I do not have any active codespaces, yet my storage usage is at 75%.

I saw some old threads talking about this being a bug in the past, I wonder if it has been reintroduced?

**Comment by searge** [comment]
Same here.
Haven't used Codespaces in months. And in fact, I stopped using it precisely because Storage usage was growing so fast. Literally, you run Codespace a few times and you already go beyond 15 GB, which is just ridiculous. Since then, I have been receiving these stupid letters on a regular basis. 

I deleted all the containers a long time ago, and it's empty.
![зображення](https://github.com/community/community/assets/2125311/7f20fe70-5709-42dd-ae3c-2c059ec9ba62)

**Comment by searge** [comment]
Hi, @am-IRh 

Could you add labels, please?
Sure, if you can.

All neighbor questions have its labels:
![image](https://github.com/community/community/assets/2125311/eab11366-7a90-463a-ba3e-279ade4c564f)
